packages/common/mui/src/lib/banner/banner.tsx

```diff
@@ -0,0 +1,78 @@
+import { Box, styled, useTheme } from '@mui/material';
+import { TokenIcon } from '@notional-finance/icons';
+import { Link } from 'react-router-dom';
+import { FormattedMessage, MessageDescriptor } from 'react-intl';
+import { H2, Paragraph } from '../typography/typography';
+import { Button } from '../button/button';
+
+interface BannerProps {
+  link?: string;
+  messages?: {
+    promptText: MessageDescriptor;
+    buttonText: MessageDescriptor;
+  };
+  title?: MessageDescriptor;
+  callback?: () => void;
+  tokenSymbol?: string;
+  buttonSuffix?: string;
+}
+
+export const Banner = ({
+  link,
+  messages,
+  title,
+  buttonSuffix,
+  callback,
+  tokenSymbol,
+}: BannerProps) => {
+  const theme = useTheme();
+  return (
+    <EmptyNoteStaking>
+      <Box sx={{ display: 'flex', alignItems: 'center' }}>
+        {tokenSymbol && <TokenIcon symbol={tokenSymbol} size={'xxl'} />}
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: 'column',
+            marginLeft: theme.spacing(3),
+          }}
+        >
+          <H2>
+            <FormattedMessage {...title} />
+          </H2>
+          <Paragraph msg={messages?.promptText} sx={{ flex: 1 }} />
+        </Box>
+      </Box>
+      {!callback && link && messages?.buttonText && (
+        <Link to={link}>
+          <Button variant="contained" size="large">
+            <FormattedMessage {...messages?.buttonText} />
+            {buttonSuffix}
+          </Button>
+        </Link>
+      )}
+    </EmptyNoteStaking>
+  );
+};
+
+const EmptyNoteStaking = styled(Box)(
```

Comment:
Since this is in Mui should this be renamed like `BannerContainer` or something like that